test/augmentation/test_augmentation.py

```diff
@@ -3062,3 +3063,16 @@ def test_smoke(self, device, dtype):
         aug = RandomRGBShift(p=1.0).to(device)
         out = aug(img)
         assert out.shape == (1, 3, 4, 5)
+
+
+class TestRandomSnow:
+    def test_smoke(self, device, dtype):
+        img = torch.rand(1, 3, 4, 5, device=device, dtype=dtype)
+        aug = RandomSnow(p=1.0).to(device)
+        out = aug(img)
+        assert out.shape == (1, 3, 4, 5)
+
+    def test_onnx_export(self, device, dtype):
+        img = torch.rand(1, 3, 4, 5, device=device, dtype=dtype)
+        aug = RandomSnow(p=1.0).to(device)
+        torch.onnx.export(aug, img, "temp.onnx", export_params=True)
```

Comment:
Also wanted to note that this onnx test does not pass since, during conversion from his to rgb, kornia uses torch.addcmul (https://github.com/kornia/kornia/blob/master/kornia/color/hls.py#L142) which is not supported for onnx. 

Wanted to discuss whether it is necessary for it to pass here and possible solutions. I think the easiest solution would be to implement the hls-rgb conversion using onnx compatible functions, but I guess there are better work arounds.

Reply:
I see, to not block this PR let's comment this test and let's put as a task/issue to upgrade `rgb_to_hls`. I'm seeing that the code in that function can be improved based on new torch functionality. /cc @cceyda might be helpful here

Reply:
That sounds good. It might just be a matter of updating according to this: https://github.com/rwightman/pytorch-image-models/issues/785#issuecomment-1074538512 `torch.addcmul`  is supported after 1.11. In anycase I'll look for an alternative for version compat.